README.adoc

```diff
@@ -871,7 +871,7 @@ Client.where(
 
 === `find` [[find]]
 
-Favor the use of `find` over `where.take!`, `find_by!`, and `find_by_id!` when you need to retrieve a single record by primary key id and raise `ActiveRecord::RecordNotFound` when the record is not found.
+Prefer the use of `find` over `where.take!`, `find_by!`, and `find_by_id!` when you need to retrieve a single record by primary key id and raise `ActiveRecord::RecordNotFound` when the record is not found.
```

Comment:
Just thinking aloud.
Does it make sense to use passages like "Prefer to use", wouldn't just "Use" do?

Reply:
Possibly? That should be a wider discussion, as similar language is used in the Ruby Style Guide.

Reply:
Actually, `the use of` does seem unnecessary, I will trim that.